datafusion/sqllogictest/test_files/string_view.slt

```diff
@@ -726,7 +726,7 @@ EXPLAIN SELECT
 FROM test;
 ----
 logical_plan
-01)Projection: overlay(CAST(test.column1_utf8view AS Utf8), Utf8("foo"), Int64(2)) AS c1
+01)Projection: overlay(test.column1_utf8view, Utf8View("foo"), Int64(2)) AS c1
 02)--TableScan: test projection=[column1_utf8view]
 
```

Comment:
Likewise, can we please also add actually running these queries to the tests

like
```sql
query 
SELECT OVERLAY(column1_utf8view PLACING 'foo' FROM 2 ) as c1
```

?

Reply:
Done! https://github.com/apache/datafusion/pull/11968/commits/907e27e976b94d26dd2ef96409f1fb7c28724e3c